Formula/cloudformation-guard.rb

```diff
@@ -0,0 +1,33 @@
+class CloudformationGuard < Formula
+  desc "Checks CloudFormation templates for compliance using a declarative syntax"
+  homepage "https://github.com/aws-cloudformation/cloudformation-guard"
+  url "https://github.com/aws-cloudformation/cloudformation-guard/archive/v0.7.0.tar.gz"
+  sha256 "f285c76ca32ca671380f254d0ce01dc6e5e3d91ef7d6e4c7fc2d6abeb7520e79"
+  license "Apache-2.0"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "./cfn-guard"
```

Comment:
```suggestion
    cd "cfn-guard" do
      system "cargo", "install", *std_cargo_args
    end
```

Reply:
ah great this is way better. thanks

Reply:
👍  (also please squash the commits into one :) )

Reply:
rebased and squashed